web/src/main/java/ua/com/vertex/logic/UserLogicImpl.java

```diff
@@ -28,6 +29,8 @@
 @PropertySource("classpath:application.properties")
 public class UserLogicImpl implements UserLogic {
     private static final Logger LOGGER = LogManager.getLogger(UserLogicImpl.class);
+    public static final String FILE_TYPE = "Invalid file type!";
```

Comment:
after '!' should be 'stupid motherfuker' or 'bitch' at least, but we can'e act this way to our customers, so can you please remove exclamation mark?

Reply:
Corrected